client/src/pages/Courses/editables/TextField.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { Box, Text, FormControl, FormLabel, Input, Divider } from "@chakra-ui/core";
+import { colors as c } from "../../../colors";
+
+const TextField = ({ isRequired = false, name, title, value, isEditing, onChange }) => {
+
+    return (
+
+        !isEditing
```

Comment:
Can we leave a comment as to what `isEditing` means and why it causes differences?

Reply:
Yup sure 😃